src/Http/Routing/test/UnitTests/EndpointRoutingMiddlewareTest.cs

```diff
@@ -61,52 +61,6 @@ public async Task Invoke_OnCall_WritesToConfiguredLogger()
             Assert.Equal(expectedMessage, write.State?.ToString());
         }
 
-        [Fact]
-        public async Task Invoke_BackCompatGetRouteValue_ValueUsedFromEndpointFeature()
```

Comment:
These were testing interop between RouteData on the IRoutingFeature and IRouteValuesFeature. These might still be valid but we need to make a decision about RouteData first.